html-block-and-inline/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title> HTML Block and Inline</title>
+</head>
+
+<body>
+  <h1>Blue Americana Snuggie&reg; Blanket</h1>
+
```

Comment:
I would recommend avoiding extra whitespace in the HTML, these can be put one line after the other